packages/components/icons/src/WorkflowsIcon.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+  IconVariant,
+  generateComponentWithVariants,
+} from '@contentful/f36-icon';
+import { generateIconComponent } from '@contentful/f36-icon/src';
```

Comment:
Do we really want to import from `/src` here?

Reply:
No!